src/net_processing.cpp

```diff
@@ -3563,32 +3563,48 @@ void ProcessMessage(
     return;
 }
 
+/** Maybe disconnect a peer and discourage future connections from its address.
+ *
+ * @param[in]   pnode           The node to check.
+ * @return                      True if the peer will be disconnected.
```

Comment:
What about something along the lines of "True if the peer was marked for disconnection in this function"?

Reply:
Sure. Happy to make that change if I need to retouch this PR.

Reply:
Agree with @dongcarl. Maybe "True if the peer is marked for disconnection in this function."

Reply:
done